### ISSUE TITLE
Current ergm@master causes fatal error on windows

### ISSUE THREAD

**Issue by smjenness** [statnet/ergm#23]
Any model estimation with the master branch of ergm causes R to crash with a call to the `ergm::ergm` on Windows. For example, this simple model:

```r
nw <- network.initialize(100, directed = FALSE)
fit <- ergm(nw ~ edges, target.stats = c(50))
```

With `debug` I have pinpointed the crash to this fx within `ergm`:
```r
	nw.stats <- summary(formula.no, response = response, 
			term.options = control$term.options)
```

**Comment by smjenness** [comment]
Yes - have you tried running the current ergm master through `devtools::win_builder`. I bet it would show these errors in R CMD Check tests

**Comment by krivit** [comment]
Well, that was subtle. Fixed in 1f5fd033d18f5661b799f28449725e981204ff69 .

**Comment by martinamorris** [comment]
context specific case sensitivity...  thx for finding/fixing
